pkgs/by-name/bo/bootloadhid/package.nix

```
@@ -0,0 +1,48 @@
+{
+  fetchurl,
+  lib,
+  libusb-compat-0_1,
+  stdenv,
+  ...
+}:
+
+stdenv.mkDerivation {
+  pname = "bootloadhid";
+  version = "0-unstable-2012-12-08";
+
+  src = fetchurl {
+    url = "https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz";
+    hash = "sha256-FU5+OGKaOi7sLfZm7foe4vLppXAY8X2fD48GTMINh1Q=";
+  };
+
+  nativeBuildInputs = [
+    libusb-compat-0_1
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    cd commandline/
+    make
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    install bootloadHID $out/bin
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "USB boot loader for AVR microcontrollers with at least 2 kB of boot load section, e.g. ATMega8";
+    homepage = "https://www.obdev.at/products/vusb/bootloadhid.html";
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [ thetaoofsu ];
+    mainProgram = "bootloadHID";
+    platforms = lib.platforms.unix;
```

Doesn't build on darwin
```suggestion
    platforms = lib.platforms.linux;
```

---

I finally managed to get a Mac VM up and running so I could fix this. It initially complained about not finding gcc, but this should now be resolved by using cc/Clang instead when building on Mac, and it builds for me. Can you confirm if it's fixed on your end?

---

It works, thanks!